test/e2e/validation_test.go

```diff
@@ -373,33 +379,36 @@ func (tc *testContext) runJob(name string, command []string) (string, error) {
 	return logs, nil
 }
 
-// getWinServices returns a map of Windows services from the instance with the given address, the key:value format being
-// name:status
-func (tc *testContext) getWinServices(addr string) (map[string]string, error) {
-	// This command returns CR+newline separated quoted CSV entries consisting of service name and status. For example:
-	// "kubelet","Running"\r\n"VaultSvc","Stopped"
-	command := "Get-Service | Select-Object -Property Name,Status | ConvertTo-Csv -NoTypeInformation"
+// getWinServices returns a map of Windows services from the instance with the given address, the map key being the
+// service's name
+func (tc *testContext) getWinServices(addr string) (map[string]winService, error) {
+	// This command returns CR+newline separated quoted CSV entries consisting of service name, state and description.
+	// For example: "kubelet","Running","OpenShift managed kubelet"\r\n"VaultSvc","Stopped",
+	command := "Get-CimInstance -ClassName Win32_Service | Select-Object -Property Name,State,Description | " +
+		"ConvertTo-Csv -NoTypeInformation"
```

Comment:
This command returns all Windows services and `testExpectedServicesRunning()` only checks Windows services managed by WMCO (i.e. `windows.RequiredServices`). 

To reduce the result set, consider adding a filter in this command to return only the services that contain the `windows.ManagedTag` in the description field.

Reply:
Good point @jrvaldes